app/controllers/foreman_ansible/api/v2/hostgroups_controller_extensions.rb

```diff
@@ -17,7 +17,7 @@ module HostgroupsControllerExtensions
           before_action :find_ansible_roles, :only => [:assign_ansible_roles]
 
           api :POST, '/hostgroups/:id/play_roles',
-              N_('Plays Ansible roles on a hostgroup')
+              N_('Runs all Ansible roles')
```

Comment:
Here the `on a hostgroup` should stay, as it refers to a host group (we could leave out all here).